### PR TITLE
Zanderz/dokan 2016.09.07

### PR DESCRIPTION
**Pull request by zanderz**
Today's test channel build, v1.0.17-20160908172133+f09996a, is based on this. Before releasing this driver update, I made another client build to test the auto update case, and found a handful of issues that are now fixed with this PR. Part of todays' efforts involved moving our Dokan build to ci.keybase.io, which works now also.

As soon as this is merged, automatic nightly windows builds can be turned on, which @cjb says should automatically go in the test channel each night.

@maxtaco @taruti

### CODE REVIEW

File: go/tools/dokanclean/main.go [hunk 1]
```diff
@@ -80,8 +80,10 @@ func removeKeybaseStartupShortcuts() {
 // If not just listing, execute each uninstaller we find and merge return codes.
 func findDokanUninstall(list bool, wow64 bool) (result int) {
 	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	// Assume this is build 32 bit, so we need this flag to see 64 but registry
```
Comment (maxtaco) [inline go/tools/dokanclean/main.go]: We spoke about it on Tuesday --- is there any way to mark dokan installs as belonging to keybase and the then only remove those that we installed?  If it's not in this build, can you make a JIRA story for it so we don't forget?  The sooner it's in the better.

**Comment by maxtaco** [comment]
@taruti can you look too?

I put a 👍 but please make that Jira story.

File: go/tools/dokanclean/main.go (hunk 1 above)
Comment (taruti) [inline go/tools/dokanclean/main.go]: A TODO to the comment so it is more easily greppable?

**Comment by taruti** [comment]
LGTM with the comment.

File: packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs [hunk 2]
```diff
@@ -254,7 +259,7 @@
     <!-- check for GDR/LDR version. See https://github.com/dokan-dev/dokany/commit/9782a6491775ef3660a71f40f381d3f48e511f14 -->
     <bal:Condition
       Message="Microsoft patch KB3033929 is not installed. A reboot is needed between the installation of said patch and the installation of Keybase. https://support.microsoft.com/en-us/kb/3033929">
-      <![CDATA[Installed OR VersionNT > v6.1 OR (VersionNT = v6.1 AND ((WINTRUSTVERSION >= v6.1.7601.18741 AND WINTRUSTVERSION <= v6.1.7601.22000) OR WINTRUSTVERSION >= v6.1.7601.22947))]]>
```
Comment (zanderz) [inline packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs]: This tweak was requested + accepted by Dokany in my pr to their installer

**Pull request merged**